tests/fixtures/request-invalid-trailer.txt

```diff
@@ -0,0 +1,8 @@
+GET / HTTP/1.1
+content-type: application/octet-stream
+transfer-encoding: chunked
+trailer: x-invalid
+
+0
+x-invalid: å

```

Comment:
I'm confused. Why is the trailer header invalid?

Reply:
The value is non-ascii, which is an invalid header.

Reply:
Oh yea! Totally slipped my mind